kafka-connect-jms/src/main/scala/com/datamountaineer/streamreactor/connect/jms/config/JMSSettings.scala

```diff
@@ -80,9 +82,9 @@ object JMSSettings extends StrictLogging {
     val destinationSelector = DestinationSelector.withName(config.getString(JMSConfigConstants.DESTINATION_SELECTOR).toUpperCase)
 
     val extraProps = config.getList(JMSConfigConstants.EXTRA_PROPS)
-                            .map(p => p.split("=").grouped(2)
-                            .map { case Array(k: String, v: String) => k.trim -> v.trim }.toMap)
-                            .toList
+      .map(p => p.split("=").grouped(2)
+        .map { case Array(k: String, v: String) => k.trim -> v.trim }.toMap)
+      .toList
```

Comment:
If you were to make PR into open source library with this one, the changes to the formatting might be unacceptable :) ...

Reply:
(there's multiple places where only formatting has changed, - I'm not marking all of them..)